src/Resources/views/formatter/badge.html.twig

```diff
@@ -0,0 +1,20 @@
+{% set classes = '' %}
+{% if background_color_hex is defined and background_color_hex is not null %}
+    {% set classes = classes ~ ' bg-[' ~ background_color_hex ~ ']' %}
+{% endif %}
+
+{% if background_color_class is defined and background_color_class is not null %}
+    {% set classes = classes ~ ' ' ~ background_color_class %}
+{% endif %}
+
+{% if link != '' %}
+    <a href="{{ link }}">
+        <div class="whatwedo-utility-badge .whatwedo-utility-badge--{{ type }}{{ classes }}">
```

Comment:
Can't have a div inside an a. But the classes directly on the `<a>`, make it an `inline-block`.